web/src/lib/static/admin/findDeadLinks.ts

```diff
@@ -293,7 +293,7 @@ export const findDeadLinks = async (): Promise<Record<string, string[]>> => {
               if (
                 result.broken &&
                 !isTemplateLink(result.url.original) &&
-                result.url.original !== "https://twitter.com/i/lists/1148625562895826945"
+                result.url.original !== "https://www.facebook.com/BusinessNJgov"
```

Comment:
[pebble] Can we reference the config here?

Reply:
yes, will fix it!